.npmignore

```diff
@@ -18,6 +18,8 @@ demo/**/*.scss
 demo/**/*.html
 demo/**/*.md
 
+dist/es5.js
```

Comment:
This is not correct. Please update file to look like 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.npmignore.temp